bazaar/rpc/src/lib.rs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2019 Alain Brenzikofer
+// This file is part of Encointer
+//
+// Encointer is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Encointer is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Encointer.  If not, see <http://www.gnu.org/licenses/>.
+
+use jsonrpc_core::{Error, ErrorCode, Result};
+use jsonrpc_derive::rpc;
+use sp_api::{Decode, Encode, ProvideRuntimeApi};
+use sp_blockchain::HeaderBackend;
+use sp_runtime::{generic::BlockId, traits::Block as BlockT};
+use std::sync::Arc;
+
+use encointer_bazaar_rpc_runtime_api::BazaarApi as BazaarRuntimeApi;
+use encointer_primitives::{communities::CommunityIdentifier, bazaar::BusinessIdentifier};
+use encointer_primitives::bazaar::{OfferingData, BusinessData};
+
+#[rpc]
+pub trait BazaarApi<BlockHash, AccountId>
+    where AccountId: 'static + Encode + Decode + Send + Sync, {
+    #[rpc(name = "bazaar_getOfferings")]
```

Comment:
Do you intentionally only mark get_businesses with `#[rpc]`? And did you confuse the name?

I have not studied the internals of the macro closely, but I believe you need to add the attribute to each function you want to expose as rpc.